messages/resume_scratch.md

```diff
@@ -4,9 +4,11 @@ Resume the creation of an incomplete scratch org.
 
 # description
 
-When the original "<%= config.bin %> org create scratch" command either times out or is run with the --async flag, it displays a job ID.
+When the original "<%= config.bin %> org create scratch" command either times out or is run with the --async flag, it
+displays a job ID.
 
-Run this command by either passing it a job ID or using the --use-most-recent flag to specify the most recent incomplete scratch org.
+Run this command by either passing it a job ID or using the --use-most-recent flag to specify the most recent incomplete
+scratch org.
```

Comment:
@WillieRuemmele , ditto.

Reply:
weird linting did that automagically - fixed now